src/bootstrap.ts

```diff
@@ -0,0 +1,31 @@
+import { Action, ActionCreator } from 'redux'
+import { ThunkAction } from 'redux-thunk'
+import { getCompanyInfo } from './search'
+import { API, Company, GlobalState } from './types'
+
+export interface CompanySymbolsReceiveAction extends Action { type: typeof COMPANY_SYMBOLS_RECEIVED_ACTION, companySymbols: Company[] }
+type ThunkResult<R> = ThunkAction<R, GlobalState, API, CompanySymbolsReceiveAction>
+
+
+const defaultCompany = {
+  name: 'Apple Inc',
+  symbol: 'aapl'
+}
+
+export const COMPANY_SYMBOLS_RECEIVED_ACTION = 'COMPANY_SYMBOLS_RECEIVED_ACTION'
+
+const receiveCompanySymbolsAction: ActionCreator<CompanySymbolsReceiveAction> = (companySymbols: Company[]): CompanySymbolsReceiveAction => ({
```

Comment:
I don't think you need to specify the return type of the function if you already specified the `ActionCreator` for the function.

Reply:
Forgot to remove that